packages/unity-interface/IUnityInterface.ts

```diff
@@ -98,6 +98,7 @@ export interface IUnityInterface {
   /** @deprecated send it with the kernelConfigForRenderer instead. */
   SetSceneDebugPanel(): void
   ShowFPSPanel(): void
+  SetLocalDCLVersion(version: string): void
```

Comment:
This PR should also contain the newer unity-renderer version including that extra function otherwise the merge will not work. Also consider that the release of unity-renderer rollout should be at 100% before merging this Pr

Reply:
Yeah this is one of the things I wanted to ask, I'm still quite ignorant regarding how the kernel works and what is the correct workflow when pushing changes on kernel/unity-render that require the other